src/runtime_src/core/tools/common/TestRunner.cpp

```diff
@@ -407,13 +407,27 @@ std::string
 TestRunner::findXclbinPath( const std::shared_ptr<xrt_core::device>& _dev,
                 boost::property_tree::ptree& _ptTest)
 {
+  auto xclbin_name = _ptTest.get<std::string>("xclbin", "");
+  std::string xclbin_path;
+#ifdef _WIN32
+  boost::ignore_unused(_dev);
+  try {
+    xclbin_path = xrt_core::environment::xclbin_path(xclbin_name);
```

Comment:
Instead of using the ifdefs here could you just use the try catch? The catch can do the legacy method of getting the xclbin path.